src/Entity/LineItem.php

```diff
@@ -45,13 +45,10 @@ class LineItem implements JsonSerializable, EntityInterface
     /** @var DateTimeInterface */
     private $endAt;
 
-    /** @var Infrastructure */
-    private $infrastructure;
-
     /** @var int */
     private $maxAttempts = 0;
 
-    public function getId(): int
+    public function getId(): ?int
```

Comment:
Why it is nullable?

Reply:
It's coming from `OAT\SimpleRoster\Entity`EntityInterface`, it is nullable because doctrine entities are mutable objects. This is what I tried to fix in the new `LtiInstance` entity (few comments below)